shared/transforms/shared_xccdf2table-stig.xslt

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cdf="http://checklists.nist.gov/xccdf/1.1" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+
+<!-- setting the external variable $notes identifies a file from which to add notes,
+	 but set to nothing by default -->
+<xsl:param name="notes" select="''" />
+
+<xsl:variable name="notegroup" select="document($notes)/notegroup" />
+
+	<xsl:template match="/">
+		<html>
+		<head>
+			<title>Rules In <xsl:value-of select="/cdf:Benchmark/cdf:title" /><xsl:if test="$notes"> with Notes for Transition to RHEL 7 Consensus</xsl:if></title>
+		</head>
+		<body>
+			<br/>
+			<br/>
+			<div style="text-align: center; font-size: x-large; font-weight:bold">
+			Rules In <i><xsl:value-of select="/cdf:Benchmark/cdf:title" /></i><xsl:if test="$notes"> with Notes for Transition to RHEL 7 Consensus</xsl:if>
```

Comment:
And once again here -- replace RHEL 7 in ```with Notes for Transition to RHEL 7 Consensus``` string with specific product name for that product.